Add information how to see the features of the dataset (#2627)
adap/flower: datasets/doc/source/how-to-use-with-tensorflow.rst

Use with TensorFlow
===================

Let's integrate ``flwr-datasets`` with ``TensorFlow``. We show you three ways how to convert the data into the formats
that ``TensorFlow``'s models expect.  Please note that, especially for the smaller datasets, the performance of the
following methods is very close. We recommend you choose the method you are the most comfortable with.

Create a ``FederatedDataset``::

  from flwr_datasets import FederatedDataset

  fds = FederatedDataset(dataset="cifar10", partitioners={"train": 10})
  partition = fds.load_partition(0, "train")
  centralized_dataset = fds.load_full("test")

> Determine the names of the features::

Reviewer: ```suggestion
Inspect the names of the features::
```

  partition.features

In case of CIFAR10, you should see the following output.

.. code-block:: none

  {'img': Image(decode=True, id=None),
  'label': ClassLabel(names=['airplane', 'automobile', 'bird', 'cat', 'deer', 'dog',
  'frog', 'horse', 'ship', 'truck'], id=None)}

> Let's move to the transformations.

Reviewer: ```suggestion
We will use the keys in the partition features in order to construct a `tf.data.Dataset <https://www.tensorflow.org/api_docs/python/tf/data/Dataset>_`. Let's move to the transformations.
```

Reviewer: I left the "Let's move to the transformations." sentence but the doc page never shows how to apply transforms

Author: I mean the transformation of the format from Dataset to NumPy (not in the sense of PyTorch transformations). But I'm fine with changing it or expressing it more clearly

Reviewer: oh, got it

TensorFlow Tensors
------------------

> Transform the data to the ``TensorFlow Tensors`` (it's not the TensorFlow dataset)::

Reviewer: ```suggestion
Transform the data to the TensorFlow `tf.Tensor <https://www.tensorflow.org/api_docs/python/tf/Tensor>`_ (it's not the TensorFlow dataset)::
```

Reviewer: better to point to docs?

Author: It's a good reference
